src/openrct2/ride/Track.cpp

```diff
@@ -1679,20 +1868,23 @@ static money32 track_remove(uint8 type, uint8 sequence, sint16 originX, sint16 o
     }
 
     money32 price = RideTrackCosts[ride->type].track_price;
-    if (ride_type_has_flag(ride->type, RIDE_TYPE_FLAG_FLAT_RIDE)) {
+    if (ride_type_has_flag(ride->type, RIDE_TYPE_FLAG_FLAT_RIDE))
+    {
         price *= FlatRideTrackPricing[type];
     }
-    else {
+    else
+    {
         price *= TrackPricing[type];
     }
     price >>= 16;
-    price = (price + cost) / 2;
+    price         = (price + cost) / 2;
```

Comment:
?

Reply:
needs fixing

Reply:
Fixed, but this does not disappear for some reason.